src/nagios_sql.py

```diff
@@ -398,6 +399,55 @@ def availability_group_status(host, user, password):
 
     return {'code': code, 'msg': msg}
 
+@nagios_test
+def synchronization_databases_ag(host, user, password, server_primary):
+    """Databases Availability group Status"""
+	
+    msg = ''
+    primary = False    

```

Comment:
Remove the extra spaces after False here.
```suggestion
    primary = False
```